.github/CODEOWNERS

```diff
@@ -0,0 +1,51 @@
+#***********************************************************************
+#*                   GNU Lesser General Public License
+#*
+#* This file is part of the GFDL Flexible Modeling System (FMS).
+#*
+#* FMS is free software: you can redistribute it and/or modify it under
+#* the terms of the GNU Lesser General Public License as published by
+#* the Free Software Foundation, either version 3 of the License, or (at
+#* your option) any later version.
+#*
+#* FMS is distributed in the hope that it will be useful, but WITHOUT
+#* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+#* for more details.
+#*
+#* You should have received a copy of the GNU Lesser General Public
+#* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+#***********************************************************************
+#
+# Each line is a file pattern followed by one or more owners.  The owner
+# can be specified by their @username, or an email address.
+#
+# Order is important.  The last matching pattern takes the most precedence.
+
+# These owners will be the default owners for all the files in the
+# repository.  Unless a later match is found, these owners
+# will be requested for a review when a PR is opened.
+*          @wrongkindofdoctor @thomas-robinson
+
+# Specific component directories
+/affinity        @bensonr
+/block_control   @bensonr
+/data_override   @GFDL-Eric
+/diag_manager    @thomas-robinson @underwoo
+/fv3gfs          @bensonr
+/libFMS          @underwoo
+
+# GNU autotools files
+Makefile.am      @underwoo
+/configure.ac    @underwoo
+/m4/             @underwoo
+*.m4             @underwoo
+
+# Files specific to GitHub or GitLab
+/.github/        @underwoo
+/.gitlab/        @underwoo
+
+# Testing files
+/.gitlab-ci.yml  @uramirez8707 @underwoo
```

Comment:
This would probably be good to loop in @mlee03 since she will be working with setting up CI.